bin/kafka-metadata-shell.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.shell.MetadataShell "$@"
```

Comment:
The shell basically becomes a public api with this. I thought I recalled that we were going to do a separate KIP? An alternative would be to locate this under `shell/bin`. Or maybe we can print a message when the tool starts out which emphasizes that this is an experimental tool without any compatibility guarantees.

Reply:
Or maybe an alternative is to put this under bin/metadata

Reply:
Actually, looks like this is documented in the KIP..